cmd/nerdctl/run_network_linux_test.go

```diff
@@ -513,26 +513,19 @@ func TestSharedNetworkStack(t *testing.T) {
 
 	containerNameJoin := testutil.Identifier(t) + "-nework"
 	defer base.Cmd("rm", "-f", containerNameJoin).AssertOK()
-	cmd := base.Cmd("run",
+	base.Cmd("run",
 		"-d",
 		"--name", containerNameJoin,
 		"--network=container:"+containerName,
 		testutil.CommonImage,
-		"sleep", "infinity")
-	cmd.AssertOK()
-
-	base.Cmd("exec", containerNameJoin, "wget", "-qO-", "http://127.0.0.1:80").
```

Comment:
I think we should not remove the get test here

Reply:
why?